lib/reporters/mini.js

```diff
@@ -163,6 +164,21 @@ class MiniReporter {
 				}
 
 				status += '  ' + colors.title(test.title) + '\n';
+
+				if (test.logs) {
+					test.logs.forEach(log => {
+						const logLines = indentString(colors.log(log), 6);
+						const logLinesWithLeadingFigure = logLines.replace(
+							/^ {6}/,
+							`    ${colors.information(figures.info)} `
+						);
```

Comment:
Oh! No that's fine, I misread the code.

I suppose the replacement string could be a constant, but on the other hand it's clearer if it's near the indentation values. So hey :)